### PR TITLE
changed class to className so the compiler will stop complaining

### PR DESCRIPTION
**Pull request by neveland98**

### CODE REVIEW

**Comment by jaitken** [comment]
Thank you for doing this lol

**Review by jaitken — APPROVED** [review]
Was annoying me too

File: src/components/ProductPhotos/productPhotos.jsx [hunk 1]
```diff
@@ -29,16 +29,16 @@ class ProductPhotos extends React.Component {
 
     render() {
         const photosList = this.state.productPhotos.map(productPhoto => {
-            return (
-                <img class="galleryPhoto" src={productPhoto.photoURL} alt="" onClick={()=>this.changeMainPicture(productPhoto.photoURL)}/>
+            return(
+                <img key={productPhoto.id} className="galleryPhoto" src={productPhoto.photoURL} alt="" onClick={()=>this.changeMainPicture(productPhoto.photoURL)}/>
```
Comment (FeiYGH) [inline src/components/ProductPhotos/productPhotos.jsx:33]: Quick question, what's the key={productPhoto.id} for?

**Review by FeiYGH — COMMENTED** [review]
Hey Nathaniel, thank you so very much! Yea those errors are annoying and this'll make it a lot better!

One comment on my ProductPhotos class. what's the key={productphoto.id} for?

**Review by FeiYGH — APPROVED** [review]
lgtm

**Pull request merged**